### ISSUE TITLE
Can not have numbered headers if table of contents is off

### ISSUE THREAD

**Issue by salomvary** [michaelrsweet/htmldoc#405]
Hey @michaelrsweet, thanks for this excellent piece of software!

I noticed this small thing which might be a bug.

While this command results in numbered headings on the pages:
```
htmldoc --numbered  -f test.pdf test.html
```

<img width="205" alt="Screenshot 2020-12-11 at 14 23 48" src="https://user-images.githubusercontent.com/193923/101908543-84a18380-3bbc-11eb-9e28-703525dca380.png">

..this does not:
```
htmldoc --no-toc --numbered  -f test.pdf test.html
```

<img width="200" alt="Screenshot 2020-12-11 at 14 23 36" src="https://user-images.githubusercontent.com/193923/101908556-89fece00-3bbc-11eb-8d15-1e007ac20710.png">

Looks like TOC is mandatory if one wants numbered headers but maybe it should not be?

```
$ htmldoc --version
1.9.10
```

Using macOS 10.14.6 and Preview.app.

**Comment by michaelrsweet** [comment]
@salomvary Um, are you sure you have those commands in the right order?

Anyways, will investigate as the two are not supposed to be connected...

**Comment by salomvary** [comment]
> @salomvary Um, are you sure you have those commands in the right order?

Uh oh, sorry I mixed them up while copypasting. Edited the description above.

**Comment by salomvary** [comment]
This is the full sequence of commands that resulted in the second screenshot that has no numbered headings:

```
$ echo '<h1>one</h1><h2>two</h2>' > test.html
$ htmldoc --no-toc --numbered  -f test.pdf test.html
PAGES: 2
BYTES: 75476
$ open test.pdf 
```

**Comment by michaelrsweet** [comment]
[master c3e825b] Fix `--numbered` with `--no-toc` (Issue #405)

**Comment by salomvary** [comment]
Thanks a lot for the quick fix @michaelrsweet! \o/
